stable_plugins/classical_ml/data_preparation/wu_palmer.py

```diff
@@ -498,16 +515,19 @@ def calculation_task(self, db_id: int) -> str:
 
         for ent in load_entities(entities_data, mimetype):
             if hasattr(ent, "_asdict"):  # is NamedTuple
+                ent_attributes: tuple[str, ...] = ent._fields
+                ent_tuple = type(ent)
```

Comment:
maybe move this into the inner if deserializer is None

and please copy the logic from ensure_array for the `isinstance(item, EntityTupleMixin)` case